website/cue/reference/components/sinks/base/databend.cue

```diff
@@ -0,0 +1,388 @@
+				description: """
+					The maximum size of a batch that will be processed by a sink.
+
+					This is based on the uncompressed size of the batched events, before they are
+					serialized / compressed.
+					"""
+				required: false
+				type: uint: {
+					default: 10000000
+					unit:    "bytes"
+				}
+			}
+			max_events: {
+				description: "The maximum size of a batch before it is flushed."
+				required:    false
+				type: uint: unit: "events"
+			}
+			timeout_secs: {
+				description: "The maximum age of a batch before it is flushed."
+				required:    false
+				type: float: {
+					default: 1.0
+					unit:    "seconds"
+				}
+			}
+		}
+	}
+	compression: {
+		description: """
+			Compression configuration.
+
+			All compression algorithms use the default compression level unless otherwise specified.
+			"""
+		required: false
+		type: string: {
+			default: "gzip"
+			enum: {
+				gzip: """
+					[Gzip][gzip] compression.
+
+					[gzip]: https://www.gzip.org/
+					"""
+				none: "No compression."
+				zlib: """
+					[Zlib]][zlib] compression.
+
+					[zlib]: https://zlib.net/
+					"""
+			}
+		}
+	}
+	database: {
+		description: "The database that contains the table that data will be inserted into."
+		required:    false
+		type: string: {
+			default: "default"
+			examples: ["mydatabase"]
+		}
+	}
+	encoding: {
+		description: "Transformations to prepare an event for serialization."
+		required:    false
+		type: object: options: {
+			except_fields: {
+				description: "List of fields that will be excluded from the encoded event."
+				required:    false
+				type: array: items: type: string: {}
+			}
+			only_fields: {
+				description: "List of fields that will be included in the encoded event."
+				required:    false
+				type: array: items: type: string: {}
+			}
+			timestamp_format: {
+				description: "Format used for timestamp fields."
+				required:    false
+				type: string: enum: {
+					rfc3339: "Represent the timestamp as a RFC 3339 timestamp."
+					unix:    "Represent the timestamp as a Unix timestamp."
+				}
+			}
+		}
+	}
+	endpoint: {
+		description: "The endpoint of the Databend server."
+		required:    true
+		type: string: examples: ["http://localhost:8000"]
+	}
+	request: {
+		description: """
+			Middleware settings for outbound requests.
+
+			Various settings can be configured, such as concurrency and rate limits, timeouts, etc.
+			"""
+		required: false
+		type: object: options: {
+			adaptive_concurrency: {
+				description: """
+					Configuration of adaptive concurrency parameters.
+
+					These parameters typically do not require changes from the default, and incorrect values can lead to meta-stable or
+					unstable performance and sink behavior. Proceed with caution.
+					"""
+				required: false
+				type: object: options: {
+					decrease_ratio: {
+						description: """
+																The fraction of the current value to set the new concurrency limit when decreasing the limit.
```

Comment:
Seems like we have some extra horizontal whitespace in this file 😅

Reply:
I believe that's the case for _all_ generated cue files, they're valid just wonky

Reply:
Actually, I see this is coming from the generation so ignore me. Seems like something we need to fix at a deeper level.

Reply:
> I believe that's the case for _all_ generated cue files, they're valid just wonky

Indeed, just realized that :)